gf-webapp-admin/src/component/rooms/room/apartmentType/ApartmentType.tsx

```diff
@@ -0,0 +1,47 @@
+import React, {useState} from "react";
+import {IRoom} from "../../../../model/room";
+import {saveRoom} from "../../../../service/room";
+import {ROOM_TYPES} from "../../../../utils/consts";
+import {EditBtn} from "../editBtn/EditBtn";
+
+
+export const ApartmentsType = ({room}:{room:IRoom}) => {
```

Comment:
add component interface instead of props description

Reply:
Done